distribution/lib/Standard/Base/0.0.0-dev/src/Data/Ordering/Natural_Order.enso

```diff
@@ -29,7 +29,7 @@ compare text1 text2 case_sensitivity=Case_Sensitivity.Sensitive =
     ## check if a single character is between '0' and '9'
     ascii_code_zero = 48
     ascii_code_nine = 57
-    is_digit=character -> character>=ascii_code_zero && character<=ascii_code_nine
+    is_digit = character -> character>=ascii_code_zero && character<=ascii_code_nine
```

Comment:
This change is covered by [following test](https://github.com/enso-org/enso/pull/3826#discussion_r1002676540), but easier to fix the syntax than wait for the test to be fixed.